src/main/java/ratspeak/Duke.java

```diff
@@ -0,0 +1,43 @@
+package ratspeak;
+
+import ratspeak.parser.CommandProcessor;
+
+public class Duke {
+

```

Comment:
You can consider adding Javadoc comments to describe the purpose of this class, otherwise I think Duke class looks good!